conda/e3sm_diags_env_dev.yml

```diff
@@ -4,38 +4,42 @@ channels:
   - defaults
 dependencies:
   # Base
+  # =================
+  - python>=3.9
```

Comment:
@chengzhuzhang I've updated the Python version from 3.8 to 3.9 since there are no longer packages conflicts related to `flake8-isort`. This should not affect the build.

Reply:
Yep, great! I was about to say that worked for me, too.